contracts/LockedNORI.sol

```diff
@@ -244,31 +244,33 @@ contract LockedNORI is
   }
 
   /**
-   * @notice Unwrap BridgedPolygonNORI tokens and makes them available for use in the BridgedPolygonNORI contract
+   * @notice Claim unlocked tokens.
    *
-   * @dev This function burns `amount` of wrapped tokens and withdraws them to the corresponding {BridgedPolygonNORI}
-   * tokens.
+   * @dev This function burns `amount` of {LockedNORI} and transfers `amount`
+   * of {BridgedPolygonNORI} from the {LockedNORI} contract's balance to
+   * `_msgSender()`'s balance.
    *
-   * Burns unlocked tokens from sender and sends them to *recipient*
+   * Enforcement of the availability of wrapped and unlocked tokens
+   * for the `_burn` call happens in `_beforeTokenTransfer`
    *
    * ##### Requirements:
    *
    * - Can only be used when the contract is not paused.
    */
-  function withdrawTo(address recipient, uint256 amount)
+  function claim(uint256 amount)
```

Comment:
The implication is that we can only withdraw to the owner and not to a different owner or contract right?